source/FunkinLua.hx

```diff
@@ -1002,20 +1002,6 @@ class FunkinLua {
 			}
 			return boobs;
 		});
-		Lua_helper.add_callback(lua, "noteTweenAngle", function(tag:String, note:Int, value:Dynamic, duration:Float, ease:String) {
```

Comment:
I haven't caught up with 0.6+ Funkin' Lua, is there any reason why this callback is omitted?

Reply:
Yes
> remove a duplicate lua function

Reply:
Since when is this a duplicate?

Reply:
idk since a couple of months at least